### PR TITLE
Useful information added to README.md file

### LINKED ISSUE #7: Information missing from README
README.md currently does not contain:
- Getting Started section that serves as a guide to start the development environment .
- Contribution section , that refers to the Contribution.md file.
These sections need to be introduced to readme when a Vagrantfile and contribution files are added.

### PR DESCRIPTION
**Pull request by idan1356**
resolves #7 
-new Contribution section to refer to contribution document
-new Getting Started section to provide information on setting up the environment for development
-Added links to GitHub repositories of contributors

### CODE REVIEW

**Review by shirazyomtov — APPROVED** [review]
LGTM

File: README.md [hunk 1]
```diff
@@ -6,15 +6,44 @@
...
+- Clone the HighTeach repository Using the command `git clone https://github.com/redhat-beyond/HighTeach.git`
+- Open command prompt in HighTeach directory
+- Spin the environment using `vagrant up`
+- Go to http://localhost:8000 in your web browser
+
+### Additional important Virtual environment commands:
+
+- You can enter the virtual machine by running `vagrant ssh`
+- Use `Vagrant halt` to shut the environment down
+- When finished, run `vagrant destroy -f` to tear down the environment
```
Comment (natifridman) [inline README.md:32]: Why do you need to use `-f` flag here?

**Review by natifridman — COMMENTED** [review]
Nice work, added one comment

File: README.md (hunk 1 above)
Comment (idan1356) [reply README.md:32]: Hi, The -f flag is not required, will remove it from the code block

Comment (idan1356) [reply README.md]: done

**Review by natifridman — APPROVED** [review]
LGTM. Please squash the commits so we can merge it

**Pull request merged**